versions.properties

```diff
@@ -23,7 +23,7 @@ version.androidx.localbroadcastmanager=1.1.0
 
 version.androidx.recyclerview=1.3.2
 
-version.androidx.core=1.8.0
+version.androidx.core=1.9.0
```

Comment:
Updated only to 9 due to https://app.asana.com/0/488551667048375/1206287150356144/f